packages/costa/src/runtime.ts

```diff
@@ -14,10 +14,15 @@ import {
   CondaConfig
 } from './index';
 
-import request from 'request-promise';
+import { get } from 'request-promise';
 import Debug from 'debug';
 
 const debug = Debug('costa.runtime');
+const UserAgent = [
```

Comment:
Why do we want to include user-agent to identify browser here

Reply:
We can remove it now :)